sdks/java/extensions/protobuf/src/main/java/org/apache/beam/sdk/extensions/protobuf/ProtoSchemaTranslator.java

```diff
@@ -157,6 +157,7 @@ static Schema getSchema(Class<? extends Message> clazz) {
 
   static Schema getSchema(Descriptors.Descriptor descriptor) {
     Set<Integer> oneOfFields = Sets.newHashSet();
+    Map<Integer, Field> oneOfFieldLocationMap = Maps.newHashMap();
```

Comment:
Nit: don't include the name of the type in the variable name.
```suggestion
    Map<Integer, Field> oneOfFieldLocation = Maps.newHashMap();
```